distributor-api-v1/operations.md

```diff
@@ -187,89 +187,89 @@ Preferred initial call used to obtain all static data about distributor configur
 | `DefaultCurrencyCode` | string | required | Default enterprise currency code in ISO 4217 format. |
 | `DefaultLanguageCode` | string | required | Default enterprise language in ISO format. |
 | `DefaultRateCurrencyCode` | string | required | Default enterprise rate currency code in ISO 4217 format. |
-| `Description` | [LocalizedText](operations.md#localizedtext) | required | Enterprise description. |
+| `Description` | [Localized text](./operations.md#localized-text) | required | Enterprise description. |
 | `Email` | string | required | Email of the enterprise. |
 | `IanaTimeZoneIdentifier` | string | required | IANA time zone identifer. |
 | `ImageId` | string | optional | Unique identifier of the enterprise logo. |
 | `IntroImageId` | string | optional | Unique identifier of the enterprise intro image. |
-| `Name` | [LocalizedText](operations.md#localizedtext) | required | Enterprise name. |
-| `Pricing` | string [Pricing](operations.md#pricing) | required | Pricing method used by the enterprise. |
-| `PrivacyPolicyUrl` | [LocalizedText](operations.md#localizedtext) | required | Enterprise privacy policy URL. |
-| `Products` | array of [Product](operations.md#product) | required | Array of active products which can be offered to the customer. |
+| `Name` | [Localized text](./operations.md#localized-text) | required | Enterprise name. |
+| `Pricing` | string [Pricing method](./operations.md#pricing-method) | required | Pricing method used by the enterprise. |
+| `PrivacyPolicyUrl` | [Localized text](./operations.md#localized-text) | required | Enterprise privacy policy URL. |
+| `Products` | array of [Product](./operations.md#product) | required | Array of active products which can be offered to the customer. |
 | `TaxEnvironmentCode` | string | required | Tax environment code. |
 | `Telephone` | string | required | Telephone of the enterprise. |
-| `TermsAndConditionsUrl` | [LocalizedText](operations.md#localizedtext) | required | Enterprise terms and conditions URL. |
+| `TermsAndConditionsUrl` | [Localized text](./operations.md#localized-text) | required | Enterprise terms and conditions URL. |
 
 #### Address
 
 | Property | Type |  | Description |
 | :--- | :--- | :--- | :--- |
 | `City` | string | optional | City. |
-| `CountryCode` | string | optional | ISO 3166-1 code of the [Country](operations.md#country). |
+| `CountryCode` | string | optional | ISO 3166-1 code of the [Country](./operations.md#country). |
 | `Latitude` | number | optional | The latitude. |
 | `Longitude` | number | optional | The longitude. |
 | `Line1` | string | optional | First address line. |
 | `Line2` | string | optional | Second address line. |
 | `PostalCode` | string | optional | Postal code. |
 
-#### Pricing
+#### Pricing method
 
 * `Gross` - The enterprise shows amount with gross prices.
 * `Net` - The enterprise shows amount with net prices.
 
-## Get Hotel Info <a id="get-hotel-info"></a>
+## Get hotel
```

Comment:
⛏️ Maybe:
```suggestion
## Get hotels
```

since the endpoint is `hotels/get` ? Some places have it too like voucher or reservationGroups/create so not sure which way we want to go.